server/src/alias/model.rs

```diff
@@ -111,14 +107,14 @@ where
                 instance,
                 path
             )
-            .fetch_optional(&mut transaction)
+            .fetch_optional(&mut *transaction)
             .await?
             .map(|result| result.uuid_id as i32)
             .ok_or(operation::Error::NotFoundError)?
         }
     };
 
-    let uuid = Uuid::fetch_via_transaction(id, &mut transaction).await?;
+    let uuid = Uuid::fetch_via_transaction(id, &mut *transaction).await?;
 
     transaction.commit().await?;
```

Comment:
Is there actually something to commit to the database?

Reply:
doesn't hurt either? Might be also faster since there is no rollback in the mysql...